ONSdigital/sdc-responses-dashboard
# Add loading screen

# Motivation and Context
Adds a loading screen to allow the user to understand the request is being processed.

# What has changed
New CSS and Js to make loader work.

# How to test?
- Add a delay in reporting or mock services to test the loader.
- Ensure loader is only visible on the first load. Subsequent AJAX call will not display loader in the middle of the screen.
- Survey top bar (title etc) only displays when rm-reporting request is finished.
